server/webapp/WEB-INF/rails.new/spec/controllers/api_v3/admin/pipelines_controller_spec.rb

```diff
@@ -669,7 +669,7 @@
     def expected_data_with_validation_errors
       {
         enable_pipeline_locking: false,
-        errors: {label_template: ["Invalid label. Label should be composed of alphanumeric text, it should contain the builder number as ${COUNT}, can contain a material revision as ${<material-name>} of ${<material-name>[:<number>]}, or use params as \#{<param-name>}."]},
+        errors: {label_template: ["Invalid label '%s'. Label should be composed of alphanumeric text, it should contain the builder number as ${COUNT}, can contain a material revision as ${<material-name>} of ${<material-name>[:<number>]}, or use params as \#{<param-name>}."]},
```

Comment:
Should be the actual string at this point, right? Also, can you please change the original error message from `it should contain the build number ...` to `it can contain the build number ...` ?